azurelinuxagent/common/cgroupapi.py

```diff
@@ -510,6 +510,8 @@ def create_cgroup(controller):
                 stderr.truncate(0)
 
                 # Try invoking the process again, this time without systemd-run
+                logger.info('Extension invocation using systemd failed, falling back to regular invocation '
```

Comment:
We're already logging systemd failure (L502), why do we need another info log for it?

Reply:
To make it more clear from the logs that we're in the fallback logic branch of this if; the logs I was examining yesterday were pretty confusing and I couldn't tell if we were firing it or not without going through the code.